lms/static/sass/elements/_controls.scss

```diff
@@ -403,3 +403,22 @@
       margin-bottom: none;
     }
   }
+
+  .btn-link {
```

Comment:
Thanks for sharing this! Instead of adding in these styles would it be possible to extend %btn-pl-secondary-base from this view? It is fine I think to have .btn-link extend this secondary base class. 

If possible it would also be good to avoid the !important overrides here, which might be accomplished by using the pl-secondary-base button extend. 

Reference to what the secondary button looks like: http://ux.edx.org/#buttons
![image](https://cloud.githubusercontent.com/assets/2023680/9204521/d631c76a-402b-11e5-95e5-db082c8971a0.png)